_summaries/niemeyer2020giraffe_1.md

```diff
@@ -1,19 +1,116 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: niemeyer2020giraffe_1
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: chengchunhsu
+score: 9/10
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## What is the core idea?
+
+* <u>Problem:</u> content creation needs to be controllable
```

Comment:
Maybe explain a bit what is controllable? And why do we need controllable? And why other methods are not controllable?